legipy/common.py

```diff
@@ -61,7 +61,7 @@ def parse_date(string):
 
     try:
         month = 1 + MONTHS.index(match.group(2))
-    except:
+    except (IndexError, ValueError, TypeError):
```

Comment:
it looks like only a `ValueError` can appear here, am I missing something ? Apart from that the PR looks good to me

Reply:
Thanks, I'm not a Python expert.
In the docs it says that `IndexError` can be raised https://docs.python.org/3/library/re.html#re.match.group
What do you think?

Reply:
In theory yes, but here the regex has matched (we verify that a few lines before), also it could be the case for a dynamic regex but here the number of groups catched is constant (4) since the regex is constant.

Also, another guess is that nobody is gonna run this library via `pypy3` since it's not cpu-intensive but it can be interesting to support the option

Reply:
Ok good, code updated.